backend/src/timed/proxy.js

```diff
@@ -80,6 +83,93 @@ function checkAccess(req) {
   return false
 }
 
+/**
+ * Send confirmation mail for order
+ * @author Jonas Cosandey (jonas.cosandey@adfinis-sygroup.ch)
+ */
+async function sendMail(req) {
+  function parseDjangoDuration(duration) {
+    let re = new RegExp(/^(-?\d+)?\s?(\d{2}):(\d{2}):(\d{2})(\.\d{6})?$/)
+
+    let [, days, hours, minutes, seconds, microseconds] = duration
+      .match(re)
+      .map(m => Number(m) || 0)
+
+    return moment.duration({
+      days,
+      hours,
+      minutes,
+      seconds,
+      milliseconds: microseconds * 1000
+    })
+  }
+
+  let mailTransporter = req.app.get('mailTransporter')
+  mailTransporter.verify(async function(error) {
+    if (error) {
+      console.log(`Mail transporter failed: ${error}`)
+      return
+    }
+
+    let { host, prefix } = config.services.timed
+    let {
+      data: {
+        id: projectId,
+        attributes: {
+          name: projectName,
+          'purchased-time': projectPurchasedTime,
+          'spent-time': projectSpentTime
+        }
+      }
+    } = JSON.parse(
+      await rp({
+        method: 'get',
+        uri: `${host}${prefix}/subscription-projects/${
+          req.body.data.relationships.project.data.id
+        }`,
+        headers: {
+          accept: 'application/vnd.api+json',
+          'content-type': 'application/vnd.api+json',
+          Authorization: `Bearer ${req.session.timedToken}`
+        }
+      })
+    )
+
+    let customer = req.session.timedCustomer,
+      duration = parseDjangoDuration(req.body.data.attributes.duration),
+      newTime = parseDjangoDuration(projectPurchasedTime)
+        .add(duration)
+        .subtract(parseDjangoDuration(projectSpentTime))
+
+    mailTransporter.sendMail({
+      from: config.mail.from,
+      to: config.mail.to,
```

Comment:
Wouldn't it make sense to let this be an array of addresses?

Reply:
`config.mail.to` can be a array or a comma separated list of addresses.